src/lib/components/PublishButton/SubmitReviewModal.js

```diff
@@ -70,14 +70,17 @@ export class SubmitReviewModal extends Component {
                       control={Checkbox}
                       fieldPath="acceptAccessToRecord"
                       label={
-                        <Trans
-                          defaults="The '{{communityTitle}}' curators will have access to
-                 <bold>view</bold> and <bold>edit</bold> your
-                 upload's metadata and files."
-                          values={{
-                            communityTitle,
+                        <div
+                          dangerouslySetInnerHTML={{
+                            __html: i18next.t(
```

Comment:
I am not sure how translations are done and the difference between methods (e.g. using the component `<Trans>` versus `i18next.t()`), so this might be a dummy question: is this change equivalent in terms of translations?

Reply:
From the Trans component documentation:

_While <Trans> gives you a lot of power by letting you interpolate or translate complex React elements, the truth is: in most cases you don't even need it.

As long you have no React/HTML nodes integrated into a cohesive sentence (text formatting like strong, em, link components, maybe others), you won't need it - most of the times you will be using the good old t function._

It is true that in this case we are using bold in the component, which may lead as to think that Trans should be used in this case. Still, as we are needing to dangerouslySetInnerHTML in order to display the special characters, we don't need to use Trans.